Source/carthage/CopyFrameworks.swift

```diff
@@ -24,11 +24,11 @@ public struct CopyFrameworksCommand: CommandType {
 				|> flatMap(.Concat) { frameworkPath -> SignalProducer<(), CarthageError> in
 					let frameworkName = frameworkPath.lastPathComponent
 
-					let source = NSURL(fileURLWithPath: frameworkPath, isDirectory: true)!
+					let source = Result(NSURL(fileURLWithPath: frameworkPath, isDirectory: true), failWith: CarthageError.InvalidArgument(description: "Could not find framework \"\(frameworkName)\" at path \(frameworkPath). Ensure that the given path is appropriately entered and that your \"Input Files\" have been entered correctly."))
 					let target = frameworksFolder().map { $0.URLByAppendingPathComponent(frameworkName, isDirectory: true) }
 
-					return SignalProducer(result: target &&& validArchitectures())
-						|> flatMap(.Merge) { (target, validArchitectures) -> SignalProducer<(), CarthageError> in
+					return combineLatest(SignalProducer(result: source), SignalProducer(result: target), SignalProducer(result: validArchitectures()))
```

Comment:
After checking out antitypical/Result#47 and #547, I swapped this back to a `combineLatest` to handle the next value coming along. Using `&&&` created a tuple within a tuple instead of a tuple with three values. 

Not sure if this is best! @robrix @ikesyo